prow/cmd/plank/main.go

```diff
@@ -128,8 +131,20 @@ func main() {
 		logrus.WithError(err).Fatal("Error creating plank controller.")
 	}
 
+	pjClientset, err := o.kubernetes.ProwJobClientset(cfg().PodNamespace, o.dryRun)
+	if err != nil {
+		logrus.WithError(err).Fatal("Failed to create prowjob client set")
+	}
+	informerFactory := prowjobinformer.NewSharedInformerFactory(pjClientset, 15*time.Second)
+	pjController := kube.NewPJController(informerFactory.Prow().V1().ProwJobs(), pjClientset)
+	stop := make(chan struct{})
+	defer close(stop)
+	go informerFactory.Start(stop)
```

Comment:
We should use `informerFactory.Start(interrupts.Context().Done())`

Reply:
Done.